### PR TITLE
skip cleanup so we don't erase babel and stuff

### PR DESCRIPTION
**Pull request by acinader**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ## Parse Server Changelog
 
 ### master
-[Full Changelog](https://github.com/parse-community/parse-server/compare/4.0.0...master)
+[Full Changelog](https://github.com/parse-community/parse-server/compare/4.0.1...master)
+
+### 4.0.1
+- FIX: correct 'new' travis config to properly deploy.  See [#6452](https://github.com/parse-community/parse-server/pull/6452)
```
Comment (dplewis) [inline CHANGELOG.md]: https://github.com/parse-community/parse-server/pull/6439 was just merged. Want to include it?

**Comment by davimacedo** [comment]
I don't think it is since no tests were changed. I understand that the error already existed. We are only improving the feedback.

**Comment by acinader** [comment]
got it.  changing.

**Comment by acinader** [comment]
@dplewis @davimacedo boo. travis still not happy.

I wont be able to look at this this weekend, but I can definately get on it on monday. I don't think there's any reason that it has to go out before that.

**Pull request merged**